### DISCUSSION TITLE
SEO stats

### DISCUSSION THREAD

**Discussion by nokia6290** [schultek/jaspr#197] [Q&A]
Hello,

Thank you for creating a new tool that tries to solve Flutter web SEO issue!

I'm curious has anyone has production ready with multiple articles and some SEO ratings ? Just asking is it 100$ identical to javascript websites in terms of SEO power.

**Comment by schultek** [comment]
> Just asking is it 100$ identical to javascript websites in terms of SEO power.

Yes. You can use all the strategies of normal SEO with Jaspr, since it compiles down to pure html/css/js.
